src/components/SubscriptionCard.js

```diff
@@ -476,29 +476,27 @@ class SubscriptionCard extends React.Component {
                     />
                   </CustomToggle>
                   <Dropdown.Menu className="menu-item">
-                    {subscription.status !== 'PENDING' && (
-                      <Fragment>
-                        <MenuItem
-                          style={menuItemStyle}
-                          eventKey={this.stateConstants.editPaymentMethod}
-                        >
-                          <FormattedMessage
-                            id="subscription.menu.editPaymentMethod"
-                            defaultMessage="Update payment method"
-                          />
-                        </MenuItem>
-                        <MenuItem
-                          style={menuItemStyle}
-                          eventKey={this.stateConstants.editAmount}
-                        >
-                          <FormattedMessage
-                            id="subscription.menu.editAmount"
-                            defaultMessage="Update amount"
-                          />
-                        </MenuItem>
-                        <MenuItem style={{ margin: '2px' }} divider />
-                      </Fragment>
-                    )}
+                    {subscription.status !== 'PENDING' && [
+                      <MenuItem
```

Comment:
The `<MenuItem/>` lacks a key prop, this can cause re-rendering issues and it is generating a warning in dev:

```
index.js:2178 Warning: Each child in an array or iterator should have a unique "key" prop.

Check the render method of `SubscriptionCard`. See https://fb.me/react-warning-keys for more information.
    in MenuItem (created by SubscriptionCard)
    in SubscriptionCard (created by InjectIntl(SubscriptionCard))
...
```